src/shared/components/event-start/event-start-utils.ts

```diff
@@ -0,0 +1,30 @@
+import { Router } from "@angular/router";
```

Comment:
Move checkTaskInEventNotRequired  method into the guard. Because we have sessionStorageService etc ideally should not be exposed to util.ts